core/templates/dev/head/css/oppia.css

```diff
@@ -1047,11 +1047,76 @@ md-card.oppia-gallery-tile {
   width: 80%;
 }
 
-.oppia-alert-blocks {
-  left: 20%;
+.oppia-toast-container {
   position: fixed;
-  width: 60%;
-  z-index: 3000;
+  z-index: 999999;
+}
+.oppia-toast-container * {
+  -moz-box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+.oppia-toast-container > div {
+  position: relative;
```

Comment:
Can you alphabetize these? Otherwise, hard to scan and see what the value of a particular property is.